clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/lifecycle/LaunchFailureNotificationCleanupAgent.java

```diff
@@ -116,6 +115,7 @@ public void run() {
           AmazonCloudProvider.ID,
           entityRef.getAccountId(),
           entityRef.getRegion(),
+          EntityTagger.ENTITY_TYPE_SERVER_GROUP,
```

Comment:
we now need to supply the `entityType` (previously it was always explicitly `servergroup`).